samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
Need to actually add a scheme claim first :)  Should we always add an identity with authtype = "AspNet.AuthenticationScheme", and a single claim with type = AuthenticationScheme, "schemeName"?